src/components/extra/Snow.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import React, { useLayoutEffect, useRef, useState } from 'react'
+import times from 'lodash/times'
+import each from 'lodash/each'
+import throttle from 'lodash/throttle'
+
+export const isSnowTime = () => {
+  const now = new Date()
+
+  return process.env.SNOW_EVENT || (now.getMonth() === 11 && now.getDate() > 21)
+}
+
+function randomBetween(min, max, round) {
+  const num = Math.random() * (max - min + 1) + min
+
+  if (round) {
+    return Math.floor(num)
+  }
+  return num
+}
+
+function Flake(x, y) {
+  const maxWeight = 4
+  const maxSpeed = 3
+
+  this.x = x
+  this.y = y
+  this.r = randomBetween(0, 1)
+  this.a = randomBetween(0, Math.PI)
+  this.aStep = 0.01
+
+  this.weight = randomBetween(2, maxWeight)
+  this.alpha = this.weight / maxWeight
+  this.speed = (this.weight / maxWeight) * maxSpeed
+
+  this.update = () => {
+    this.x += Math.cos(this.a) * this.r
+    this.a += this.aStep
+    this.y += this.speed
+  }
+}
+
+type Props = {
+  numFlakes: number,
+}
+
+const Snow = ({ numFlakes }: Props) => {
+  const canvasRef = useRef()
+  const containerRef = useRef()
+
+  const [dimension, setDimension] = useState({ width: 0, height: 0 })
+
+  useLayoutEffect(() => {
+    const ro = new ResizeObserver(
+      throttle(() => {
+        setDimension(containerRef.current.getBoundingClientRect())
+      }),
+      250,
```

Comment:
i think a parenthesis was missed here, the 250 is to be a parameter of throttle i guess? because right now when you resize it's def not throttled :D

Reply:
LOL